google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java

```diff
@@ -291,6 +294,36 @@ public void testDisableLifecycleManagement() {
     assertEquals(0, storage.get(BUCKET).getLifecycleRules().size());
   }
 
+  @Test
+  public void testSetPublicAccessPreventionEnforced() {
+    SetPublicAccessPreventionEnforced.setPublicAccessPreventionEnforced(PROJECT_ID, BUCKET);
+    assertEquals(
+        storage.get(BUCKET).getIamConfiguration().getPublicAccessPrevention(),
+        BucketInfo.PublicAccessPrevention.ENFORCED);
+    PrintStream standardOut = System.out;
+    final ByteArrayOutputStream snippetOutputCapture = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(snippetOutputCapture));
+    GetPublicAccessPrevention.getPublicAccessPrevention(PROJECT_ID, BUCKET);
+    String snippetOutput = snippetOutputCapture.toString();
+    System.setOut(standardOut);
+    assertTrue(snippetOutput.contains("enforced"));
+  }
+
+  @Test
+  public void testSetPublicAccessPreventionUnspecified() {
```

Comment:
Should these tests be verifying public access prevention without the API call to `getPublicAccessPrevention`? This is testing the `setPublicAccessPreventionUnspecified` function, but also relies on the functionality of `getPublicAccessPrevention`. Not sure what practices are on how small a unit should be for each unit test.

Reply:
Fair point, I'll decouple.